prometheus/base/alerting/prometheus.pod_alerts.yaml

```diff
@@ -35,3 +35,11 @@ spec:
       annotations:
         summary: "Pod {{ $labels.pod }} is not ready"
         description: "Pod {{ $labels.pod }} in namespace {{ $labels.namespace }} is not ready for more than 15 minutes."
+    - alert: PersistentVolumeIssues
```

Comment:
I'm not sure whether pod alerts is the best location for this rule. I don't think this is a pod metric.

I think we should have it on another file dedicated for persistent volumes or something like that.

Reply:
Added and updated all the other comments you have made. Thanks @yftacherzog & @avi-biton